src/main/java/dansplugins/factionsystem/commands/MembersCommand.java

```diff
@@ -52,39 +57,72 @@ public void execute(CommandSender sender, String[] args, String key) {
         final Faction faction;
         if (args.length == 0) {
             if (!(sender instanceof Player)) {
-                sender.sendMessage(translate(getText("OnlyPlayersCanUseCommand")));
+                new PlayerService().sendMessageType(sender, getText("OnlyPlayersCanUseCommand")
+                        , "OnlyPlayersCanUseCommand", false);
                 return;
             }
             faction = getPlayerFaction(sender);
             if (faction == null) {
-                sender.sendMessage(translate("&c" + getText("AlertMustBeInFactionToUseCommand")));
+                new PlayerService().sendMessageType(sender, getText("AlertMustBeInFactionToUseCommand")
+                        , "AlertMustBeInFactionToUseCommand", false);
                 return;
             }
         } else {
             faction = getFaction(String.join(" ", args));
             if (faction == null) {
-                sender.sendMessage(translate("&c" + getText("FactionNameNotRecognized")));
+                new PlayerService().sendMessageType(sender, "&c" + getText("FactionNameNotRecognized"),
+                        Objects.requireNonNull(new MessageService().getLanguage().getString("FactionNotFound"))
+                                .replaceAll("#faction#", String.join(" ", args
+                                )), true);
                 return;
             }
         }
         // send Faction Members
-        sender.sendMessage(translate("&b----------\n" + getText("MembersOf", faction.getName())));
-        sender.sendMessage(translate("&b----------\n"));
-        faction.getMemberList().stream()
-                .map(Bukkit::getOfflinePlayer)
-                .map(player -> {
-                    String rank = "";
-                    String color = "&a";
-                    if (faction.isOfficer(player.getUniqueId())) {
-                        rank = "*";
-                        color = "&b";
-                    }
-                    if (faction.isOwner(player.getUniqueId())) {
-                        rank = "**";
-                        color = "&c";
-                    }
-                    return translate("&f" + player.getName() + color + rank);
-                }).forEach(sender::sendMessage);
-        sender.sendMessage(translate("&b----------\n"));
+        if (!new MedievalFactions().USE_NEW_LANGUAGE_FILE) {
```

Comment:
We don't want to create a new instance of the main plugin class - pass the reference into the command.